src/Linfo/Linfo.php

```diff
@@ -87,7 +87,17 @@ public function __construct($settings = array())
         $distro_class = '\\Linfo\\OS\\'.$os;
         $this->parser = new $distro_class($this->settings);
     }
-
+    
+    // Forward missing method request to the parser
+    public function __call ( $name , $args )
```

Comment:
Can you remove the whitespaces around the function arguments? eg:

    public function __call($name, $args)